server/oauth-core/src/main/java/org/eclipse/lyo/server/oauth/core/utils/AbstractAdapterCredentialsFilter.java

```diff
@@ -540,8 +579,11 @@ public void generateAccessToken(OAuthRequest oAuthRequest)
 			// For now, hard-code the consumers.
 			config.setConsumerStore(createConsumerStore());
 		} catch (Throwable t) {
-			System.err.println("Error initializing the OAuth consumer store: " +  t.getMessage());
+			log.error("Error initializing the OAuth consumer store: " +  t.getMessage());
 		}
+		
+		config.setServletUri(getServletUri());
+		log.trace("OauthConfig is working with ServletUri: {}", config.getServletUri());
```

Comment:
I like to put such notices using `.info` if they are printed only once after startup.

Reply:
fixed.